website/cue/reference/remap/functions/parse_nginx_log.cue

```diff
@@ -3,8 +3,10 @@ package metadata
 remap: functions: parse_nginx_log: {
 	category:    "Parse"
 	description: """
-        Parses Nginx access and error log lines. Lines can be in [`combined`](\(urls.nginx_combined)), or [`error`](\(urls.nginx_error)) format.
-        """
+
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ Extra new-line, I also expect this may fail the cue-fmt task - as it looks like the indentation changed. There's a make target for `cue-fmt` in the `website/` directory, if I remember correctly.
```suggestion
```

Reply:
It was failing for the first two times I pushed, now I found the reason - VScode was inserting tabulations instead of spaces, now it's fixed

Reply:
```
-> % ./scripts/check-docs.sh
Skipping cue files format validation - reserved for CI
Validating cue files correctness...
Success! The contents of the sources in the "./website/cue" directory are valid
```